README.md

```diff
@@ -36,6 +36,15 @@ If you don't have composer installed globally then another way to create a new Z
 
 If you don't have access to curl, then install Composer into your project as per the [documentation](https://getcomposer.org/doc/00-intro.md).
 
+### Installing using Vagrant
+
+This project support a basic [Vagrant](http://docs.vagrantup.com/v2/getting-started/index.html) configuration to provide with a bash script your Skeleton Application into the VirtualBox VM.  
```

Comment:
This is not installation. I think this new section should go under Web Server Setup, after CLI and before Apache.

Also, suggested word-smithing:

```
### Vagrant Server

This project supports a basic [Vagrant](http://docs.vagrantup.com/v2/getting-started/index.html) configuration with an inline shell provisioner to run the Skeleton Application in a [VirtualBox](https://www.virtualbox.org/wiki/Downloads).  
```